aspnetcore/signalr/java-client.md

```diff
@@ -20,12 +20,12 @@ The sample Java console app referenced in this article uses the SignalR Java cli
 
 ## Install the SignalR Java client package
 
-The *signalr-0.1.0-preview1-35029* JAR file allows clients to connect to SignalR hubs. To find the latest JAR file version number, see the [Maven search results](https://search.maven.org/search?q=g:com.microsoft.aspnet%20AND%20a:signalr&core=gav).
+The *signalr-0.1.0-preview1-35174* JAR file allows clients to connect to SignalR hubs. To find the latest JAR file version number, see the [Maven search results](https://search.maven.org/search?q=g:com.microsoft.aspnet%20AND%20a:signalr&core=gav).
```

Comment:
This should be `preview2` shouldn't it?

Reply:
Oh. Yes, you're right.

Reply:
Fix it in a new PR, and I'll get it merged ASAP.

Reply:
👍